releasenotes/v1.10.0-beta.0.md

```diff
@@ -0,0 +1,166 @@
+<!-- markdownlint-disable no-inline-html line-length -->
+# Changes since v1.9.0
+
+## :warning: Breaking Changes
+
+- Bump CAPI to v1.10.0-beta.0 (#2432)
+- Deprecate Metal3data templateReference (#2326)
+- Remove IPAM deployment logic (#1993)
+- Remove Metal3Machine owner reference from BMH (#1742)
+
+:rotating_light: This is a BETA RELEASE. Use it only for testing purposes.
+If you find any bugs, file an [issue](https://github.com/metal3-io/cluster-api-provider-metal3/issues/new/).
+
+<details>
+<summary>More details about the release</summary>
+
+## :sparkles: New Features
+
+- Deprecate NoCloudProvider and add CloudProviderEnabled (#2108)
+
+## :bug: Bug Fixes
+
+- Fix: pulling unused images in the middle of tests (#2271)
+- Add use of IPAMprovider to e2e tests (#2348)
```

Comment:
this should perhaps be in features?

Reply:
I agree. Also the one above. We should be more careful when reviewing so that PRs that they get the proper emojis. Any PR that only touches test code should be :seedling: